### PR TITLE
question detail screen

### LINKED ISSUE #16: Question Detail Screen

### PR DESCRIPTION
**Pull request by brukted**
closes #25

### CODE REVIEW

**Review by nawrazi — APPROVED** [review]

**Review by VariantAbdulaziz — APPROVED** [review]

**Review by Biruk-Tassew — APPROVED** [review]

File: qelem/lib/widgets/answer_card.dart [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import 'package:flutter/material.dart';
+
+class AnswerCard extends StatelessWidget {
+  const AnswerCard({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Card(
```
Comment (hello-world-was-taken) [inline qelem/lib/widgets/answer_card.dart:8]: App level card view theme might be better. What do you think?

File: qelem/lib/widgets/answer_card.dart [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+      clipBehavior: Clip.antiAlias,
+      child: Column(
+        children: [
+          const ListTile(
+            leading: Icon(Icons.arrow_drop_down_circle),
+            title: Text('Emre Varol'),
+          ),
+          Padding(
+            padding: const EdgeInsets.all(16.0),
+            child: Text(
```
Comment (hello-world-was-taken) [inline qelem/lib/widgets/answer_card.dart:18]: Align the text to left

File: qelem/lib/screens/question_detail_screen.dart [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+            SliverPadding(
+                padding: const EdgeInsets.all(0.0),
+                sliver: SliverList(
+                  delegate: SliverChildListDelegate([
+                    Padding(
+                      padding: const EdgeInsets.only(
+                          top: 23.0, left: 16.0, right: 16.0),
+                      child: Column(
+                        children: [
+                          Text(
```
Comment (hello-world-was-taken) [inline qelem/lib/screens/question_detail_screen.dart:101]: Align text to left

File: qelem/lib/screens/question_detail_screen.dart [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+                      child: Text("ANSWER",
+                          style: Theme.of(context).primaryTextTheme.button),
+                    ),
+                  ),
+                ))
+          ],
+        ),
+        headerSliverBuilder: (BuildContext context, bool innerBoxIsScrolled) {
+          return <Widget>[
+            SliverAppBar(
```
Comment (hello-world-was-taken) [inline qelem/lib/screens/question_detail_screen.dart:76]: Why did you make the app bar collapsable?

**Review by hello-world-was-taken — CHANGES_REQUESTED** [review]

File: qelem/lib/screens/question_detail_screen.dart (hunk 4 above)
Comment (brukted) [reply qelem/lib/screens/question_detail_screen.dart:76]: To utilize the screen space better when reading answers.

File: qelem/lib/screens/question_detail_screen.dart (hunk 3 above)
Comment (brukted) [reply qelem/lib/screens/question_detail_screen.dart:101]: it is.

File: qelem/lib/widgets/answer_card.dart (hunk 2 above)
Comment (brukted) [reply qelem/lib/widgets/answer_card.dart:18]: it is.

File: qelem/lib/screens/question_detail_screen.dart (hunk 3 above)
Comment (hello-world-was-taken) [reply qelem/lib/screens/question_detail_screen.dart:101]: No it is not

File: qelem/lib/widgets/answer_card.dart (hunk 2 above)
Comment (hello-world-was-taken) [reply qelem/lib/widgets/answer_card.dart:18]: No it is not.

**Review by hello-world-was-taken — APPROVED** [review]

**Pull request merged**